psydac/cad/tests/test_geometry.py

```diff
@@ -154,9 +154,24 @@ def test_geometry_1():
 #==============================================================================
 
 def teardown_module():
+    import os
     from sympy import cache
     cache.clear_cache()
 
+    # Remove HDF5 files generated by Geometry.export()
+    filenames = [
+        'geo.h5',
+        'geo_0.h5',
+        'geo_1.h5',
+        'quart_circle.h5',
+        'quart_circle_0.h5',
+        'quart_circle_1.h5',
+        'circle.h5'
+    ]
+    for fname in filenames:
+        if os.path.exists(fname):
+            os.remove(fname)
+
```

Comment:
it would be better to export the geometry files in a tmp directory and remove its content, rather than doing it for each file.

Reply:
I actually find it preferable to have each module clean after itself. This way you can run `pytest` on a single file, and the geometry files will be removed at the end even if some tests fail.